lightbar_manager/src/lightbar_manager/lightbar_manager_node.cpp

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright (C) 2018-2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include "lightbar_manager/lightbar_manager_node.hpp"
+#include <algorithm>
+
+namespace lightbar_manager
+{
+LightBarManager::LightBarManager(std::string node_name) : lbm_(node_name)
+{
+    nh_ = ros::CARMANodeHandle{"lightbar_manager"};
+    pnh_ = ros::CARMANodeHandle{"~"};
+    node_name_ = node_name;
+}
+
+void LightBarManager::turnOffAll()
+{
+    // Since lightbar_manager is guaranteed to have the highest priority, the manager is guaranteed to get all the control upon request
```

Comment:
Is this comment accurate? What if the light bar manager is not highest on the priority list?